src/components/Header.tsx

```diff
@@ -66,8 +73,8 @@ export const Header: FunctionComponent = () => {
             <div className="w-auto mb-4 lg:mb-0">
               <p className="font-medium">Current Account</p>
               <p className="break-all">
-                {wallet.walletAddress === "-" && wallet.walletAddress}
-                {wallet.walletAddress !== "-" && (
+                {wallet.walletAddress === undefined && "-"}
```

Comment:
dont need to use === undefined. Can use falsy assertion

!wallet.walletAddress && "-"

Reply:
done.